src/screens/ContactScreen/elements/ContactPreview.js

```diff
@@ -37,18 +35,11 @@ const ContactPreview = ({users = [], onPress}) => {
                     right: -5
                   }}>
                   <Pressable onPress={() => onPress(item)}>
-                    <MemoIconClose width={24} height={24} />
+                    <MemoIconClose width={24} height={24} fill={COLORS.signed_primary} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `MemoIconClose` component has been changed to `COLORS.signed_primary`. Ensure that this color change aligns with the overall design and accessibility guidelines, as it might affect visibility and user experience.
```diff
- <MemoIconClose width={24} height={24} />
+ <MemoIconClose width={24} height={24} fill={COLORS.signed_primary} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->